components/templates/RecordIndexTemplate/index.tsx

```diff
@@ -0,0 +1,36 @@
+import Button from '@components/atoms/Button'
+import ContentCardList from '@components/atoms/ContentCardList'
+import Header from '@components/Header'
+import PageLayout from '@components/layouts/PageLayout'
+import useLoadMore from '@hooks/loadMore'
+import MarkdownFileWithUrl from '@interfaces/MarkdownFileWithUrl'
+import metaDataDefaults from '@lib/metaDataDefaults'
+import { FunctionComponent } from 'react'
+
+export interface Props {
+    url: string
+    headerText: string
+    records: MarkdownFileWithUrl[]
+}
+
+const RecordIndexTemplate: FunctionComponent<Props> = props => {
+    const metaTags = { ...metaDataDefaults, title: props.headerText, url: metaDataDefaults.url.concat(props.url) }
+    const loadMoreHook = useLoadMore(props.records, 10)
+
+    return (
+        <PageLayout metaTags={metaTags}>
+            <Header
+                headerText={props.headerText}
+                isRecordIndex={true}
+            />
+            <ContentCardList records={loadMoreHook.currentRecords} />
+            <Button 
+                onClick={() => loadMoreHook.setPage(loadMoreHook.page + 10)}
+                className='primary'>
+                    Load More
```

Comment:
On tags that don't have a lot of tutorials, the `Load more` seems out of place. Do you think some logic about the number of records before showing that button would be useful?

<img width="811" alt="Screen Shot 2021-10-29 at 3 29 41 PM" src="https://user-images.githubusercontent.com/5977693/139491660-c41736ae-a5b9-4296-982e-5589a29345de.png">

Reply:
That sounds great to me & I totally agree!